architecture/core_concepts/routes.adoc

```diff
@@ -1052,3 +1053,53 @@ spec:
     in the subdomain `example.com`. `*.example.com` is the subdomain for host
     name `wildcard.example.com` to reach the exposed service.
 ====
+
+[[disable-namespace-ownership-check]]
+== Disabling Namespace Ownership Check
+
+By default, different paths to a host name (and subdomains for wildcard routes)
+can only be claimed by routes in the same namespace as the owning route.  This
+feature relaxes that route ownership policy.
+
+For exanple, if namespace ns1 creates the oldest route r1  www.abc.xyz, then it
+owns that host name and subdomain (abc.xyz) for wildcard routes.  if another
+namespace ns2 creates a route with say a different path www.abc.xyz/path1/path2,
+then it would fail because a route in another namespace (ns1 in this case) "owns"
+that host.
+
+It becomes more complex with
+xref:../../install_config/router/default_haproxy_router.html#using-wildcard-routes[wildcard routes]
+as you now also have to look at who owns "abc.xyz" (or basically has a route *.abc.xyz).
```

Comment:
Instead of `as you now also ...`, maybe `as we now need to look at routes that are granted claims to hosts (*.abc.xyz) that belong to the subdomain (abc.xyz) associated with the wildcard route`.